jscomp/bin/whole_compiler.ml

```diff
@@ -81985,9 +81993,10 @@ and ipp_comment : 'a . P.t -> 'a  -> unit = fun   f comment ->
     ]}
 *)
 
-and pp_comment f comment = 
-  if String.length comment > 0 then 
-    P.string f "/* "; P.string f comment ; P.string f " */" 
+and pp_comment f comment =
+  if String.length comment > 0 then
```

Comment:
this function does not need be changed if you change the outer most comment to be 'Nested ::'